src/Dispatcher/GroupPosBased.php

```diff
@@ -13,6 +13,9 @@ protected function dispatchVariableRoute($routeData, $uri) {
                 continue;
             }
 
+            // to resolve vimeo/psalm PossiblyUndefinedVariable
+            $i = 1;
```

Comment:
Better to use a `@psalm-suppress` docblock here, especially as it really points to [a bug in Psalm](137) (as the leftmost sub-expression of the first `for`-loop expression is always evaluated)